packages/lib/shared/src/jsonDataUri.ts

```diff
@@ -0,0 +1,5 @@
+import { stringify } from './stringify';
+
+export function createJsonDataUri(data: any): string {
```

Comment:
I think it'll make sense to name the file the name as the function including if just one function is exported